package.json

```diff
@@ -5,24 +5,37 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "type-check": "tsc"
+    "type-check": "tsc",
+    "storybook": "start-storybook -p 6006 -c .storybook",
+    "build-storybook": "build-storybook"
   },
   "dependencies": {
```

Comment:
`@storybook/addon-knobs`, `@types/storybook__react`, `awesome-typescript-loader`, and `react-docgen-typescript-loader` should be moved to dev dependencies.